tmt/steps/provision/testcloud.py

```diff
@@ -320,6 +320,15 @@ class GuestTestcloud(tmt.GuestSsh):
     _image: Optional['testcloud.image.Image'] = None
     _instance: Optional['testcloud.instance.Instance'] = None
 
+    @property
+    def is_ready(self) -> bool:
+        if self._instance is None:
+            return False
+
+        state = testcloud.instance._find_domain(self._instance.name,
+                                                self._instance.connection)
+        return state == 'running'
```

Comment:
Please, give it a few test rounds - I suggested this approach, and it seems to match the desired solution, but I did not test it.

Reply:
Very well then :)